scrabble/README.md

```diff
@@ -4,21 +4,61 @@ Determine which of two Scrabble words is worth more.
 
 ## Background
 
-TODO
+In the game of [Scrabble](https://scrabble.hasbro.com/en-us/rules), players create words to score points, and the number of points is the sum of the point values of each letter in the word. 
```

Comment:
Can we add a Markdown table of point values for each letter, just so there's something visually easy to reference?